README.md

```diff
@@ -71,3 +71,8 @@ Working agreement:
 * Regression lasts for one week and all of the regression bugs, that have been found, should be fixed with highest priority in week 6.
 * Afterwards application is to be tested and approved by the client. 
 * Sign-off from the client means that we push the code to the production after week 6.
+
+
+## Responsive desing breakpoints
+
+Angular implementation use extensively Bootstrap 4 - we will comply with [standard Bootstrap 4 breakpoints described on its website](https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints).
```

Comment:
it's website

Reply:
Agree, but I am not sure if we want to keep all 4 breakpoints. We might discuss that with @Bosmanfrx and @TrebuhD .

Reply:
@w1stler that's not `it is` - it and adjective that describe a website `of it` (see: https://pl.pons.com/t%C5%82umaczenie/angielski-polski/its)

Generally we don't need to create separate media query for extra small devices, that enough to remember about them and to see if volontulo looks sane on them.